### PR TITLE
Support deployment of topology dualtor-mixed

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
PR introduced the dualtor-mixed topology. This change is to support deployment of the new topology.

#### How did you do it?
* Updated the playbook and ansible modules for deployment of the new dualtor-mixed topology.
* Added vms-kvm-dual-mixed KVM testbed.
For the bridges created for active-active interfaces, the open flow rules are not added yet.
Need more PRs to address that after the details of the open-flow rules are sorted out.

#### How did you verify/test it?
Tested add-topo, remove-topo, restart-ptf for KVM dualtor-mixed testbed.
Tested add-topo, remove-topo, restart-ptf for physical dualtor-mixed testbed.

### CODE REVIEW

File: ansible/roles/vm_set/library/vm_topology.py [hunk 1]
```diff
@@ -184,8 +189,11 @@ def adaptive_temporary_interface(vm_set_name, interface_name, reserved_space=0):
 class HostInterfaces(object):
     """Data descriptor that supports multi-DUTs interface definition."""
 
-    def __get__(self, obj, objtype):
-        return obj._host_interfaces
+    def __set_name__(self, owner, name):
```
Comment (lolyu) [inline ansible/roles/vm_set/library/vm_topology.py:192]: `__set_name__` is introduced by Python 3.6, it doesn't work for Python2

Comment (wangxin) [reply ansible/roles/vm_set/library/vm_topology.py:192]: Nice catch! I just realized that I only tested on Ubuntu20 which uses python3.

**Review by lolyu — APPROVED** [review]
LGTM

**Pull request merged**